src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import DayPicker from 'react-day-picker'
+import 'react-day-picker/lib/style.css'
+import styled from 'styled-components/macro'
+
+const WEEKDAYS_SHORT = ['S', 'M', 'T', 'W', 'T', 'F', 'S']
+
+export default function Calendar() {
+  const modifiers = {
+    sundays: { daysOfWeek: [0] },
+  }
+  const modifiersStyles = {
+    sundays: {
+      color: '#000000',
+      backgroundColor: '#F0F0F0',
+    },
+  }
+  return (
+    <StyledCalendar>
+      <DayPicker
+        weekdaysShort={WEEKDAYS_SHORT}
+        firstDayOfWeek={1}
+        modifiers={modifiers}
+        modifiersStyles={modifiersStyles}
+      />
+    </StyledCalendar>
+  )
+}
+
+const StyledCalendar = styled.div`
+  display: flex;
+  justify-content: center;
+  font-family: Roboto;
```

Comment:
You should add a fall-back font-family, in case the webfont can't be loaded, such as `font-family: Roboto, sans-serif`. Google fonts usually gives you a suggestion in the code snippet.